src/debugpy/_vendored/pydevd/_pydevd_bundle/pydevd_vars.py

```diff
@@ -595,11 +596,15 @@ def method():
         del frame
 
 
-def change_attr_expression(frame, attr, expression, dbg, value=SENTINEL_VALUE):
+def change_attr_expression(frame, attr, expression, dbg, value=SENTINEL_VALUE, /, scope: Union[ScopeRequest, None]=None):
     """Changes some attribute in a given frame."""
     if frame is None:
         return
 
+    if scope is not None:
```

Comment:
I would've expected `if scope:` rather than `if scope is not None:`. Is that just the convention in debugpy?

Reply:
Yeah it's used a lot